src/store/configureStore.dev.js

```diff
@@ -0,0 +1,11 @@
+import { createStore, applyMiddleware, compose } from 'redux';
+import thunk from 'redux-thunk';
+import reducers from '../reducers';
+
+const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
+const configureStore = () => {
+	const store = createStore(reducers, composeEnhancers(applyMiddleware(thunk)));
+	return store;
+};
```

Comment:
```suggestion
const configureStore = () => {
	return createStore(reducers, composeEnhancers(applyMiddleware(thunk)));
};
```

saves some bites in memory

Reply:
done!